templates/email/default/_email_settings.html

```diff
@@ -106,7 +115,8 @@
 list_item_style = "padding-bottom: 20px; margin-bottom: 20px; border-bottom: $list_item_border_bottom;"
 list_item_h2_style = "margin: 0 0 16px 0; font-size: 21px; line-height: 24px;"
 list_item_p_style = "margin: 0 0 16px 0;"
-list_item_date_style = "font-size: 14px; line-height: 20px; margin: 0; color: $color_gunmetal_light;"
+list_item_p_style_dark_mode = "margin: 0 0 16px 0; color: $color_white !important;"
```

Comment:
Is this style needed now you've fixed the primary column text colour? `list_item_p_style` doesn't set a colour, so I'd hope a dark mode version wasn't needed, what do you think?

Reply:
Not a problem, so technically is not needed, however that was preventing the user name below the problem to keep having the colour: gunmetal_light which has accessibility issues with the dark background colour. 
I wasn't too in love with the user name having the same colour with the problem heading(both white), so what I did in my recent commit ([e18b833](https://github.com/mysociety/fixmystreet/pull/3835/commits/e18b83361e8ad1a9f70147ddd9fe797b0f93c74e)) was to add a class to the user name, to be able to select that element and in dark mode instead of having a white colour, now will have: color_blue_pale. That passes the contrast test and it doesn't have the same colour as the heading. It's a subtle change, but I thought it would be better than creating a new variable (new colour) just for this particular style.

Let me know what you think

Reply:
Okay, so you've fixed the username, so I guess my question still stands - is list_item_p_style_dark_mode needed any more? :)